### PR TITLE
ChatTile component + Contacts Page

### PR DESCRIPTION
**Pull request by shellyannissa**
Partially done with feature #37 .Added the landing page for Chat section .

### CODE REVIEW

**Review by dhwanish-3 — CHANGES_REQUESTED** [review]
Please
remove avatar
main.dart changes that you did for testing purpose

**Comment by shellyannissa** [comment]
Have made the requested changes. The frontend is done apart from minor changes such as mechanism to add new member for chatting, chatting with images etc...

File: lib/widgets/chat_tile.dart
Comment (dhwanish-3) [inline lib/widgets/chat_tile.dart:1]: maxlines

File: lib/main.dart
Comment (dhwanish-3) [inline lib/main.dart:1]: remove testing purpose page

**Review by dhwanish-3 — CHANGES_REQUESTED** [review]

**Comment by dhwanish-3** [comment]
Please rebase to main

**Review by dhwanish-3 — APPROVED** [review]
alrighty

**Pull request merged**